ensureChangelogHeaders.js

```diff
@@ -14,7 +14,7 @@ const fs = require("fs");
 const glob = require("glob");
 const startCase = require("lodash/startCase");
 
-glob.sync(path.join("packages/*/CHANGELOG.md")).forEach(file => {
+glob.sync(path.join("packages/*/CHANGELOG.md")).forEach((file) => {
```

Comment:
I wonder if there was a prettier config change that is adding all the extra parenthesis on cases like this.